### PR TITLE
Vanilla LFF rebel faction

### PR DESCRIPTION
**Pull request by ante185**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
Adds a Livonia-appropriate rebel faction for Vanilla.
It'd be the only vanilla faction that doesn't start with SMGs.

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

File: A3A/addons/core/Templates/Templates/Vanilla/Vanilla_Reb_LFF.sqf [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+["vehiclesTruck", ["I_G_Van_01_transport_F", "a3a_Van_02_black_transport_F", "a3a_Van_02_black_vehicle_F"]] call _fnc_saveToTemplate;
+private _vehiclesAT = ["a3a_Offroad_01_black_AT_F"];
+private _vehicleAA = [];
+
+["vehiclesBoat", ["I_C_Boat_Transport_02_F"]] call _fnc_saveToTemplate;
+
+["vehiclesPlane", ["I_C_Plane_Civil_01_F"]] call _fnc_saveToTemplate;
+
+private _vehiclesCivCar = ["C_Offroad_01_comms_F", "C_Offroad_01_covered_F","C_Offroad_01_F", "C_Hatchback_01_F", "C_Hatchback_01_sport_F", "C_SUV_01_F"];
+["vehiclesCivTruck", ["C_Van_01_transport_F", "C_Van_02_transport_F", "C_Van_02_vehicle_F"]] call _fnc_saveToTemplate;
```
Comment (Bob-Murphy) [inline A3A/addons/core/Templates/Templates/Vanilla/Vanilla_Reb_LFF.sqf:23]: Again, the vans you picked are Laws of War DLC.

**Review by Bob-Murphy — CHANGES_REQUESTED** [review]
not loaded but looks good besides the vans

File: A3A/addons/core/Templates/Templates/Vanilla/Vanilla_Reb_LFF.sqf (hunk 1 above)
Comment (ante185) [reply A3A/addons/core/Templates/Templates/Vanilla/Vanilla_Reb_LFF.sqf:23]: This is carried over from the other rebel templates, both vanilla and not, as they're not something that'd get used against the will of the player it should be fine.

**Review by Tiny-DM — APPROVED** [review]
The van thing is because every other vanilla faction also includes the Laws of War locked vans, to work as another high capacity transport option when sushi trucks are out of style. This matches the FIA template, and it might be worth having a conversation later about locking the vans behind the Laws of War checkbox. It seems reasonable if we restrict other vehicles (I want to say the M249 Jeep but I'm not sure) with similar conditions.

**Review by Bob-Murphy — APPROVED** [review]

**Pull request merged**